### ISSUE TITLE
AI Script Error

### ISSUE THREAD

**Issue by jinlan** [freeorion/freeorion#2681]
```

[21 十二月 21:57:18] AI_2: AI_Error: AI script error : FreeOrionAI.py:generateOrders():338  - Exception float division by zero while trying to issue_fleet_orders_for_fleet_missions
Traceback (most recent call last):
  File "F:/Game\fo\FreeOrion\default\python\AI\FreeOrionAI.py", line 335, in generateOrders
    action()
  File "F:/Game\fo\FreeOrion\default\python\AI\FleetUtilsAI.py", line 596, in issue_fleet_orders_for_fleet_missions
    mission.issue_fleet_orders()
  File "F:/Game\fo\FreeOrion\default\python\AI\AIFleetMission.py", line 541, in issue_fleet_orders
    fleet_portion_to_remain = self._portion_of_fleet_needed_here()
  File "F:/Game\fo\FreeOrion\default\python\AI\AIFleetMission.py", line 626, in _portion_of_fleet_needed_here
    return CombatRatingsAI.rating_needed(potential_threat, local_defenses) / float(fleet_rating)
ZeroDivisionError: float division by zero


```

-------------------------------------------------------------------------------------------
My version: c0eb3bb
[save](https://github.com/jinlan/issue/raw/master/FreeOrion_AI_Error_0080_20191221_215354.7z)

Press "end turn" error comes up in AI turn.

**Comment by Vezzra** [comment]
That is rather serious, added to release milestone.

@Morlic-fo, can you take a look at this?

**Comment by Vezzra** [comment]
Confirmed on macOS.

**Comment by Morlic-fo** [comment]
Resolved by #2682

**Comment by Vezzra** [comment]
@Morlic-fo, thanks for the quick fix!
